roachtest: asyncpg failed

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18900490?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18900490?buildTab=artifacts#/asyncpg) on master @ [c471a724f99a5add3ccefe07995251760d0e2212](https://github.com/cockroachdb/cockroach/commits/c471a724f99a5add3ccefe07995251760d0e2212):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-c471a724f99a5add3ccefe07995251760d0e2212
Tests run against asyncpg v0.24.0
213 Total Tests Run
165 tests passed
48 tests failed
34 tests skipped
16 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test_transaction.TestTransaction.test_transaction_nested (unexpected)
--- FAIL: test_transaction.TestTransaction.test_transaction_regular (unexpected)
For a full summary look at the asyncpg artifacts 
An updated blocklist (asyncpgBlocklist) is available in the artifacts' asyncpg log
test artifacts and logs in: /artifacts/asyncpg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

_Grafana is not yet available for azure clusters_</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-48041

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18900479?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18900479?buildTab=artifacts#/asyncpg) on master @ [c471a724f99a5add3ccefe07995251760d0e2212](https://github.com/cockroachdb/cockroach/commits/c471a724f99a5add3ccefe07995251760d0e2212):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-c471a724f99a5add3ccefe07995251760d0e2212
Tests run against asyncpg v0.24.0
213 Total Tests Run
165 tests passed
48 tests failed
34 tests skipped
16 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test_transaction.TestTransaction.test_transaction_regular (unexpected)
--- FAIL: test_transaction.TestTransaction.test_transaction_nested (unexpected)
For a full summary look at the asyncpg artifacts 
An updated blocklist (asyncpgBlocklist) is available in the artifacts' asyncpg log
test artifacts and logs in: /artifacts/asyncpg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=false</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18900479/asyncpg/1740206123666/1740206402625)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18906414?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18906414?buildTab=artifacts#/asyncpg) on master @ [fa8342d50c1bdfbc307f4bca9bc0d34da4475258](https://github.com/cockroachdb/cockroach/commits/fa8342d50c1bdfbc307f4bca9bc0d34da4475258):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-fa8342d50c1bdfbc307f4bca9bc0d34da4475258
Tests run against asyncpg v0.24.0
213 Total Tests Run
165 tests passed
48 tests failed
34 tests skipped
16 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test_transaction.TestTransaction.test_transaction_regular (unexpected)
--- FAIL: test_transaction.TestTransaction.test_transaction_nested (unexpected)
For a full summary look at the asyncpg artifacts 
An updated blocklist (asyncpgBlocklist) is available in the artifacts' asyncpg log
test artifacts and logs in: /artifacts/asyncpg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

_Grafana is not yet available for azure clusters_</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18906403?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18906403?buildTab=artifacts#/asyncpg) on master @ [fa8342d50c1bdfbc307f4bca9bc0d34da4475258](https://github.com/cockroachdb/cockroach/commits/fa8342d50c1bdfbc307f4bca9bc0d34da4475258):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-fa8342d50c1bdfbc307f4bca9bc0d34da4475258
Tests run against asyncpg v0.24.0
213 Total Tests Run
165 tests passed
48 tests failed
34 tests skipped
16 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test_transaction.TestTransaction.test_transaction_regular (unexpected)
--- FAIL: test_transaction.TestTransaction.test_transaction_nested (unexpected)
For a full summary look at the asyncpg artifacts 
An updated blocklist (asyncpgBlocklist) is available in the artifacts' asyncpg log
test artifacts and logs in: /artifacts/asyncpg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18906403/asyncpg/1740292722682/1740292992334)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18912532?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18912532?buildTab=artifacts#/asyncpg) on master @ [cbb292c487ea7413772a5c3c349f93467264c190](https://github.com/cockroachdb/cockroach/commits/cbb292c487ea7413772a5c3c349f93467264c190):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-cbb292c487ea7413772a5c3c349f93467264c190
Tests run against asyncpg v0.24.0
213 Total Tests Run
165 tests passed
48 tests failed
34 tests skipped
16 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test_transaction.TestTransaction.test_transaction_nested (unexpected)
--- FAIL: test_transaction.TestTransaction.test_transaction_regular (unexpected)
For a full summary look at the asyncpg artifacts 
An updated blocklist (asyncpgBlocklist) is available in the artifacts' asyncpg log
test artifacts and logs in: /artifacts/asyncpg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=false</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18912532/asyncpg/1740378566987/1740378838086)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18912543?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18912543?buildTab=artifacts#/asyncpg) on master @ [cbb292c487ea7413772a5c3c349f93467264c190](https://github.com/cockroachdb/cockroach/commits/cbb292c487ea7413772a5c3c349f93467264c190):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-cbb292c487ea7413772a5c3c349f93467264c190
Tests run against asyncpg v0.24.0
213 Total Tests Run
165 tests passed
48 tests failed
34 tests skipped
16 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test_transaction.TestTransaction.test_transaction_nested (unexpected)
--- FAIL: test_transaction.TestTransaction.test_transaction_regular (unexpected)
For a full summary look at the asyncpg artifacts 
An updated blocklist (asyncpgBlocklist) is available in the artifacts' asyncpg log
test artifacts and logs in: /artifacts/asyncpg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=false</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

_Grafana is not yet available for azure clusters_</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

looks like we are failing at:

https://github.com/MagicStack/asyncpg/blob/d0797f1f7aed2d28253ec27adac39794a09b2690/tests/test_transaction.py#L67

```
======================================================================
ERROR: test_transaction_nested (test_transaction.TestTransaction)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/data1/asyncpg/tests/test_transaction.py", line 67, in test_transaction_nested
    self.assertTrue(self.con.is_in_transaction())
AssertionError: False is not true
...
asyncpg.exceptions.InvalidSavepointSpecificationError: savepoint "__asyncpg_savepoint_756__" does not exist
```

and

```
======================================================================
ERROR: test_transaction_regular (test_transaction.TestTransaction)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/data1/asyncpg/asyncpg/_testbase/__init__.py", line 92, in wrapper
  ...
asyncpg.exceptions.DuplicateTableError: relation "defaultdb.public.mytab" already exists
```

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18923269?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18923269?buildTab=artifacts#/asyncpg) on master @ [8bd38972db32f5e683ba6dc0eb599334919919e5](https://github.com/cockroachdb/cockroach/commits/8bd38972db32f5e683ba6dc0eb599334919919e5):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-8bd38972db32f5e683ba6dc0eb599334919919e5
Tests run against asyncpg v0.24.0
213 Total Tests Run
165 tests passed
48 tests failed
34 tests skipped
16 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test_transaction.TestTransaction.test_transaction_nested (unexpected)
--- FAIL: test_transaction.TestTransaction.test_transaction_regular (unexpected)
For a full summary look at the asyncpg artifacts 
An updated blocklist (asyncpgBlocklist) is available in the artifacts' asyncpg log
test artifacts and logs in: /artifacts/asyncpg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=leader</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

_Grafana is not yet available for azure clusters_</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18923258?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18923258?buildTab=artifacts#/asyncpg) on master @ [8bd38972db32f5e683ba6dc0eb599334919919e5](https://github.com/cockroachdb/cockroach/commits/8bd38972db32f5e683ba6dc0eb599334919919e5):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-8bd38972db32f5e683ba6dc0eb599334919919e5
Tests run against asyncpg v0.24.0
213 Total Tests Run
165 tests passed
48 tests failed
34 tests skipped
16 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test_transaction.TestTransaction.test_transaction_regular (unexpected)
--- FAIL: test_transaction.TestTransaction.test_transaction_nested (unexpected)
For a full summary look at the asyncpg artifacts 
An updated blocklist (asyncpgBlocklist) is available in the artifacts' asyncpg log
test artifacts and logs in: /artifacts/asyncpg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18923258/asyncpg/1740466696437/1740466969397)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18934383?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18934383?buildTab=artifacts#/asyncpg) on master @ [c44b3134d23d6037e1d5d5e12461a6914aa588e3](https://github.com/cockroachdb/cockroach/commits/c44b3134d23d6037e1d5d5e12461a6914aa588e3):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-c44b3134d23d6037e1d5d5e12461a6914aa588e3
Tests run against asyncpg v0.24.0
213 Total Tests Run
165 tests passed
48 tests failed
34 tests skipped
16 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test_transaction.TestTransaction.test_transaction_regular (unexpected)
--- FAIL: test_transaction.TestTransaction.test_transaction_nested (unexpected)
For a full summary look at the asyncpg artifacts 
An updated blocklist (asyncpgBlocklist) is available in the artifacts' asyncpg log
test artifacts and logs in: /artifacts/asyncpg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

_Grafana is not yet available for azure clusters_</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18934372?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18934372?buildTab=artifacts#/asyncpg) on master @ [c44b3134d23d6037e1d5d5e12461a6914aa588e3](https://github.com/cockroachdb/cockroach/commits/c44b3134d23d6037e1d5d5e12461a6914aa588e3):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-c44b3134d23d6037e1d5d5e12461a6914aa588e3
Tests run against asyncpg v0.24.0
213 Total Tests Run
165 tests passed
48 tests failed
34 tests skipped
16 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test_transaction.TestTransaction.test_transaction_regular (unexpected)
--- FAIL: test_transaction.TestTransaction.test_transaction_nested (unexpected)
For a full summary look at the asyncpg artifacts 
An updated blocklist (asyncpgBlocklist) is available in the artifacts' asyncpg log
test artifacts and logs in: /artifacts/asyncpg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=epoch</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18934372/asyncpg/1740551423587/1740551695605)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18944241?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18944241?buildTab=artifacts#/asyncpg) on master @ [32fe7ee0b575276bff803469006bba08737840cf](https://github.com/cockroachdb/cockroach/commits/32fe7ee0b575276bff803469006bba08737840cf):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-32fe7ee0b575276bff803469006bba08737840cf
Tests run against asyncpg v0.24.0
213 Total Tests Run
165 tests passed
48 tests failed
34 tests skipped
16 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test_transaction.TestTransaction.test_transaction_regular (unexpected)
--- FAIL: test_transaction.TestTransaction.test_transaction_nested (unexpected)
For a full summary look at the asyncpg artifacts 
An updated blocklist (asyncpgBlocklist) is available in the artifacts' asyncpg log
test artifacts and logs in: /artifacts/asyncpg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18944241/asyncpg/1740637828295/1740638102763)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18944252?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/18944252?buildTab=artifacts#/asyncpg) on master @ [32fe7ee0b575276bff803469006bba08737840cf](https://github.com/cockroachdb/cockroach/commits/32fe7ee0b575276bff803469006bba08737840cf):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v25.2.0-alpha.00000000-dev-32fe7ee0b575276bff803469006bba08737840cf
Tests run against asyncpg v0.24.0
213 Total Tests Run
165 tests passed
48 tests failed
34 tests skipped
16 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: test_transaction.TestTransaction.test_transaction_nested (unexpected)
--- FAIL: test_transaction.TestTransaction.test_transaction_regular (unexpected)
For a full summary look at the asyncpg artifacts 
An updated blocklist (asyncpgBlocklist) is available in the artifacts' asyncpg log
test artifacts and logs in: /artifacts/asyncpg/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicBufferedSender=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

_Grafana is not yet available for azure clusters_</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/cmd/roachtest/tests/asyncpg_blocklist.go
+++ b/pkg/cmd/roachtest/tests/asyncpg_blocklist.go
@@ -59,7 +59,13 @@ var asyncpgBlocklist = blocklist{
 	`test_prepare.TestPrepare.test_prepare_28_max_args`:                                                   "unknown",
 	`test_prepare.TestPrepare.test_prepare_statement_invalid`:                                             "experimental feature - https://github.com/cockroachdb/cockroach/issues/49329",
 	`test_timeout.TestTimeout.test_timeout_06`:                                                            "unknown",
-	`test_utils.TestUtils.test_mogrify_simple`:                                                            "multi-dim arrays - https://github.com/cockroachdb/cockroach/issues/32552",
+	// The test_transaction* tests fail when attempting to check if we are in an
+	// active transaction. This occurs due to the autocommit_before_ddl behavior.
+	// The transaction they have open creates a table right after opening, which
+	// causes the transaction to close.
+	"test_transaction.TestTransaction.test_transaction_nested":  "142048",
+	"test_transaction.TestTransaction.test_transaction_regular": "142048",
+	`test_utils.TestUtils.test_mogrify_simple`:                  "multi-dim arrays - https://github.com/cockroachdb/cockroach/issues/32552",
 }
 
 var asyncpgIgnoreList = blocklist{

